### ISSUE TITLE
Allow to select key notation independent form translation. 

### ISSUE THREAD

**Issue by daschuer** [mixxxdj/mixxx#13954]
### Feature Description

There exists different notations for keys in the world. This partially overlaps with the GUI language the user uses. However Users may want to use the English GUI but use the key notation like do mi fa so they are used to. 
This requires an independent selection in the key notation preferences.

**Comment by Swiftb0y** [comment]
Can't they just use the the custom notation feature? This is quite an edge case, especially considering our users are DJs, not orchestra conductors...

**Comment by daschuer** [comment]
Yes they can. However this is a tedious work we can do only one and put it into the source code. 

In additon there is a mess around key translation in Tranifex.

**Comment by Eve00000** [comment]
But some of us had a [musical education as a child](https://mixxx.zulipchat.com/#narrow/channel/109171-development/topic/Track.20Key.20.26.20BPM.20Color.20Coding/near/445315074): 2 years notes-class & harmony before touching an instrument (except a recorder)
